guide/english/designer-tools/index.md

```diff
@@ -16,6 +16,8 @@ Great for static content.
 
 [Figma](https://www.figma.com) - [Guide](https://github.com/freeCodeCamp/guides/blob/master/src/pages/designer-tools/Figma/index.md)
 
+[Gravit Designer](https://www.designer.io/)
```

Comment:
Add the video guide for Gravit as has been done for the restof the designers, link to playlist: https://www.youtube.com/playlist?list=PLqsk_4aqUvEYbEdXeg7_sxtF4MNfleFQJ&disable_polymer=true